tensordata/scRNA.py

```diff
@@ -0,0 +1,272 @@
+from concurrent.futures import ProcessPoolExecutor
+from pathlib import Path
+
+import anndata
+import numpy as np
+import numpy.typing as npt
+import pandas as pd
+import scanpy as sc
+from scipy.sparse import csr_matrix, spmatrix
+from sklearn.utils.sparsefuncs import inplace_column_scale, mean_variance_axis
+
+DATA_DIR = Path(__file__).parent / "scRNA"
+
+
+def prepare_dataset(
```

Comment:
I'd argue that the normalization is a part of the analysis, and should go in with Pf2.

Reply:
Yeah, this also removes the dependence on sc.pp.normalize_total.